tox.ini

```diff
@@ -10,6 +10,7 @@ python =
 [tox]
 envlist =
     type-check
+    lint
```

Comment:
We also need to add this to `[gh-actions]` for `3.10` so that linting is run in CI.